modules/src/anime.py

```diff
@@ -21,7 +21,7 @@ def process(input, entities):
         text = template.get_text()
 
         template = ButtonTemplate(text)
-        template.add_web_url('Kitsu URL', 'https://kitsu.io/anime/' + top_anime['slug'])
+        template.add_web_url('Kitsu URL', 'https://kitsu.io/anime/' + top_anime['slug'],'Youtube URL', 'https://www.youtube.com/watch?v=' + top_anime['youtubeVideoId'])
```

Comment:
There should be two buttons. See https://github.com/swapagarwal/JARVIS-on-Messenger/blob/master/modules/src/music.py#L27